src/SpedreadWindow.vala

```diff
@@ -532,6 +534,19 @@ class SpedreadWindow : Gtk.ApplicationWindow {
                        SettingsBindFlags.DEFAULT
         );
 
+        _words_at_a_time = new Gtk.SpinButton (null, 25, 0);
+        _words_at_a_time.set_increments (1, 2);
+        _words_at_a_time.set_range (1,10);
+
+        settings.bind ("words-at-a-time",
+                _words_at_a_time, "value",
+                SettingsBindFlags.DEFAULT
+        );
+
+        _words_at_a_time.value_changed.connect(() => {
+            text_changed();
+            });
```

Comment:
Indentation looks a bit weird here and you forgot to add a space between `.connect` and the first parenthesis (I know it looks weird if you're not used to it but it's pretty much standard when programming in Vala ^^')